api/commands/compile.ts

```diff
@@ -27,6 +27,11 @@ export async function compile(
     throw new Error(`Compile Error: 'dataform.json' is invalid. ${e}`);
   }
 
+  // Schema overrides field can be set in two places, projectConfigOverride is the source of truth.
+  if (compileConfig.projectConfigOverride && compileConfig.projectConfigOverride.schemaSuffix) {
+    compileConfig.schemaSuffixOverride = compileConfig.projectConfigOverride.schemaSuffix;
```

Comment:
doing this sort of indicates that the plan is to keep `CompileConfig.schema_suffix_override` around long-term, is that the plan?

otherwise i'd expect the inverse; copying `compileConfig.schemaSuffixOverride` into `compileConfig.projectConfigOverride.schemaSuffix` (unless it already has a value)